### ISSUE TITLE
Some controls do not support ToolTips when the user uses the keyboard to navigate

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#4305]
**Controls that don't support tooltips when the user uses the mouse to navigate:**
- [x]  ListViewItem (#4320)
- [x]  TabPage (#2717)
- [x]  TreeNode (#4439)

**Controls that have a property for a tooltip, but it is not displayed when the user uses the mouse or keyboard to navigate:**
- [ ] ContextMenu
- [ ]  DataGridView (it is only displayed when the user hovers over a cell (for example: DataGridViewButtonCell))
- [ ]  PrintPreviewDialog
- [ ]  PropertyGrid

---

**Controls that have no tooltip properties:**
- [X] ColorDialog
- [X] FolderBrowserDialog
- [X] FontDialog
- [X] NotifyIcon
- [X] OpenFileDialog
- [X] PageSetupDialog
- [X] PrintDocument
- [X] PrintDialog
- [X] SaeFileDialog
- [X] TolStripSeparator

**Controls that cannot be focused using the keyboard (but the tooltip is displayed when the user uses the mouse):**
- [X] DataGridViewButtonColumn
- [X] DataGridViewCheckBoxColumn
- [X] DataGridViewComboBoxColumn
- [X] DataGridViewImageColumn
- [X] DataGridViewLinkColumn
- [X] DataGridViewTextBoxColumn
- [X] FlowLayoutPanel
- [X] GroupBox	
- [X] Label
- [X] MenuStrip
- [X] Panel
- [X] PictureBox	
- [X] ProgressBar
- [X] SplitterPanel
- [X] StatusStrip
- [X] ToolStrip
- [X] ToolStripLabel
- [X] ToolStripProgressBar
- [X] ToolStripStatusLabel

**Controls that support tooltips when the user uses the keyboard or mouse to navigate:**
- [X] BindingNavigator - buttons
- [X] DataGridViewButtonCell
- [X] DataGridViewCheckBoxCell
- [X] DataGridViewComboBoxCell
- [X] DataGridViewImageCell
- [X] DataGridViewLinkCell
- [X] DataGridViewTextBoxCell
- [X] DateTimePicker
- [X] DomainUpDown
- [X] ListBox
- [X] ListView
- [X] MaskedTextBox
- [X] MonthCalendar
- [X] NumericUpDown
- [X] PrintPreviewControl
- [X] PrintPreviewDialog - buttons
- [X] PropertyGrid - buttons
- [X] RadioButton
- [X] RichTextBox
- [X] SplitterContainer
- [X] TabControl
- [X] TextBox
- [X] ToolStripButton
- [X] ToolStripComboBox
- [X] ToolStripDropDownButton
- [X] ToolStripMenuItem
- [X] ToolStripSplitButton
- [X] ToolStripTextBox
- [X] TreeViewLinkLabel

**Comment by SergeySmirnov-Akvelon** [comment]
Closed because the main task of adding missing keyboard tooltips is complete.
